src/main/java/com/jvm_bloggers/entities/blog/Blog.java

```diff
@@ -64,6 +64,16 @@
     @Column(name = "ACTIVE")
     private boolean active;
 
+    @Column(name = "MODERATION_REQUIRED")
+    private Boolean moderationRequired;
```

Comment:
I would make it `nullable=false` with database value set to 
a) `false` for all existing personal blog posts
b) `true` for all existing video channels and company blogs.
Maybe this could be done when a new column is created in Liquibase script.